src/readable/block.go

```diff
@@ -1,29 +1,75 @@
 package readable
 
-import "github.com/skycoin/skycoin/src/coin"
+import (
+	"errors"
+
+	"github.com/skycoin/skycoin/src/cipher"
+	"github.com/skycoin/skycoin/src/coin"
+)
 
 // BlockHeader represents the readable block header
 type BlockHeader struct {
-	BkSeq             uint64 `json:"seq"`
-	BlockHash         string `json:"block_hash"`
-	PreviousBlockHash string `json:"previous_block_hash"`
-	Time              uint64 `json:"timestamp"`
-	Fee               uint64 `json:"fee"`
-	Version           uint32 `json:"version"`
-	BodyHash          string `json:"tx_body_hash"`
+	BkSeq        uint64 `json:"seq"`
+	Hash         string `json:"block_hash"`          // TODO -- rename "hash"
+	PreviousHash string `json:"previous_block_hash"` // TODO -- rename "previous_hash"
```

Comment:
why leave this for TODO here?

Reply:
Took them off and added them to the list in #420

Reply:
It can't be changed without breaking the API